### PR TITLE
Add heading for renderable values

### PR DESCRIPTION
**Pull request by arthurevans**
This is inspired by the changes that @bennypowers suggested in #935 ... This is basically the same thing with some changes that seemed like too fiddly to work through in GH suggestions.

### CODE REVIEW

**Comment by arthurevans** [comment]
Ping @justinfagnani @sorvell on this small change.

File: packages/lit-dev-content/site/docs/templates/expressions.md [hunk 1]
```diff
@@ -137,11 +137,11 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values
-* Sentinel values
-* `TemplateResult` objects created with the `html` function.
-* DOM nodes
-* Arrays or iterables of any of the supported types
+* Primitive values likes strings, numbers, and booleans.
+* `TemplateResult` objects created with the [`html`](/docs/api/templates/#html) or [`svg`](/docs/api/templates/#svg) functions.
```
Comment (sorvell) [inline packages/lit-dev-content/site/docs/templates/expressions.md:141]:
```suggestion
* `TemplateResult` objects created with the [`html`](/docs/api/templates/#html) or [`svg`](/docs/api/templates/#svg) functions, when inside an `svg` element.
```

Seems most important to call this out here. I'm meh on putting it in the other spot as well.

**Review by sorvell — APPROVED** [review]
Just a minor suggestion, but overall looks good to me.

**Review by justinfagnani — APPROVED** [review]

File: packages/lit-dev-content/site/docs/templates/expressions.md (hunk 1 above)
Comment (arthurevans) [reply packages/lit-dev-content/site/docs/templates/expressions.md:141]: Like the idea, but this way it's not clear that "when inside an `svg` element" applies only to the `svg` function. Maybe:

```suggestion
* `TemplateResult` objects created with the [`html`](/docs/api/templates/#html) function (or the [`svg`](/docs/api/templates/#svg) function, if the expression is inside an `<svg>` element).
```

**Pull request merged**